### DISCUSSION TITLE
openssl10 path issue

### DISCUSSION THREAD

**Discussion by sawyerlin** [midoks/mdserver-web#537] [General]
I think this should be openssl_10 but not openssl10

**Comment by sawyerlin** [comment]
https://github.com/midoks/mdserver-web/blob/master/plugins/mysql/versions/5.6/install.sh#L80

**Comment by midoks** [comment]
https://github.com/midoks/mdserver-web/blob/master/plugins/php/lib/openssl_10.sh#L25
